src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkContextAuction.tsx

```diff
@@ -1,40 +1,101 @@
 import React from "react"
-import { createFragmentContainer, graphql } from "react-relay"
-import { Header } from "../Header"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+
+import { ArtworkContextAuction_artwork } from "__generated__/ArtworkContextAuction_artwork.graphql"
+import { ArtworkContextAuctionQuery } from "__generated__/ArtworkContextAuctionQuery.graphql"
+import { ContextConsumer } from "Artsy"
+import { renderWithLoadProgress } from "Artsy/Relay/renderWithLoadProgress"
+
 import {
   ArtistArtworkGrid,
   AuctionArtworkGrid,
-  PartnerArtworkGrid,
   RelatedWorksArtworkGrid,
 } from "./ArtworkGrids"
 
-export const ArtworkContextAuction = props => {
+interface ArtworkContextAuctionProps {
+  /** The artworkID to query */
+  artworkID: string
+  /** Used to exclude the current work from the currently-shown work from grid */
+  artworkMongoID: string
```

Comment:
I'm not *thrilled* with the naming (but I am bad at naming).

Really, `artworkID` is the _slug_, and this is the actual id. But with our use of slug as id plus MP adding another Relay-specific `__id`....its kinda messy.

Reply:
(i'm not suggesting to change the naming...i think the comment explains it sufficiently).

Reply:
@mzikherman - what do you think a good name would be? Originally I had it as `_id` but that seemed less clear to an unfamiliar observer and so wanted to strive for literalness -- "a mongoid is used to exclude a given work of art from a query"

Reply:
If anything, I would be updating our existing uses of `artworkID`, (`artistID`, etc.) to possibly be `artworkSlug`, to reflect that (although those work with slugs _or_ ids).

And then something like this, which explicitly must be an id (and not a slug), can stay `excludeIDs` or whatever.

Again, totally up to you! That can be a separate/followup, for sure.

Reply:
That makes sense 👍